apps/db_pendings/import.sh

```diff
@@ -97,6 +99,7 @@ function import() {
                 echo "DROP PROCEDURE IF EXISTS \`updateDb\`;" >> "$newFile";
             fi;
 
+            COMMIT_HASH="$COMMIT_HASH $(git log --diff-filter=A "$entry" | grep "^commit " | sed -e 's/commit //')"
```

Comment:
What if a commit included more than 1 revision? It will duplicate the hash in the list?